massgov/mayflower
# React/dp 22741 global nav keyboard fix

Fixed:
  - project: React
    component: HamburgerNav
    description: A11y fix keyboard navigation. (#1478)
    issue: DP-22741
    impact: Patch

To test:
Make sure that the issues described in [the ticket](https://massgov.atlassian.net/browse/DP-22741) are fixed 
- [ ] localhost:6006/index.html?path=/story/organisms-headerhamburger--header-hamburger-example
- [ ] localhost:6006/index.html?path=/story/organisms-headerhamburger--header-hamburger-without-utility-nav
